presets/inference/llama2-chat/inference_api.py

```diff
@@ -191,6 +191,13 @@ def get_metrics():
         except Exception as e:
             return {"error": str(e)}
 
+    @app_main.get("/version")
+    def health_check():
+        with open("/workspace/llama/model_name.txt", "r") as f:
+            model_name = f.read()
+
+        return {"version": model_name}
```

Comment:
Currently version here will just print model_name, we also need the version (e.g. 0.0.3)